tests/aws/test_ses.py

```diff
@@ -422,8 +422,8 @@ def test_ses_sns_topic_integration_send_email(
         messages.sort(key=sort_mail_sqs_messages)
         snapshot.match("messages", messages)
 
-    @markers.parity.only_localstack
-    @markers.parity.aws_validated
+    @markers.aws.only_localstack
```

Comment:
that's interesting: it seems to be both: `only_localstack` + `aws.validated` 😅 
as it's a snapshot test, i guess the `only_localstack` can be removed some time :)

Reply:
yeah I'll soon be enforcing that a test can only ave one of the `@markers.aws.*` markers. Basically any test in `tests/aws/` has to have *exactly* **one** marker.